src/apps/openfluid-devstudio/WareBuildStatusWidget.hpp

```diff
@@ -74,12 +95,7 @@ class OPENFLUID_API WareBuildStatusWidget : public QWidget
 
     ~WareBuildStatusWidget();
 
-    openfluid::ui::common::ActionLabel* installLabel();
-
-    openfluid::ui::common::ActionLabel* configureModeLabel();
-
-    openfluid::ui::common::ActionLabel* jobsLabel();
-
+    void injectBuildOptions(openfluid::waresdev::WareBuildOptions BuildOptions);
```

Comment:
use "set" instead of "inject"

Reply:
I assumed "set" was reserved for pure setter functions. Here it is set + refresh visual, still applies ?